<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration/Layout Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pushButton_switch" >
       <property name="toolTip" >
        <string>Switch to a different configuration file</string>
       </property>
       <property name="text" >
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="topMargin" >
         <number>1</number>
        </property>
        <property name="bottomMargin" >
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pix" >
       <property name="text" >
        <string>Replace images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pupils" >
       <property name="text" >
        <string>Import pupils</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="QTreeWidget" name="treeWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="indentation" >
        <number>30</number>
       </property>
       <property name="uniformRowHeights" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="tableWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::CurrentChanged</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>Field Name                 </string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Value</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="undoRedoEnabled" >
       <bool>true</bool>
      </property>
      <property name="readOnly" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pushButton_clone" >
       <property name="contextMenuPolicy" >
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip" >
        <string>Make a copy of the current node</string>
       </property>
       <property name="text" >
        <string>Clone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete" >
       <property name="toolTip" >
        <string>Remove the current node</string>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save" >
       <property name="toolTip" >
        <string>Save to temporary file</string>
       </property>
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quit" >
       <property name="toolTip" >
        <string>Save the changes and exit the program</string>
       </property>
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>682</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>642</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
